cheatsheets/Vulnerable_Dependency_Management_Cheat_Sheet.md

```diff
@@ -0,0 +1,233 @@
+# Introduction
+
+The objective of the cheat sheet is to provide a proposal of approach regarding the handling of vulnerable third-party dependencies when they are detected, and this, depending on different situation.
+
+The cheat sheet is not tools oriented but it contains a *tools* section providing pointer to software (free and commercial) that can be used to detect vulerable dependencies, if possible, in a cross-technology way because it's now rare that a project use a single technology. 
+
+**Note:** 
+
+Proposal mentioned in this cheat sheet are not silver-bullet (recipes that work in all situations) but should be used a foundation and adapted to your context.
+
+# Context
+
+Most of the project use third-party dependencies to delegate handling of differents kind of operations like generation of document in a specific format, HTTP communications, data parsing of a specific format...
+
+It's a good approach because it allow the development team to focus on the real application code supporting the expected business feature. The downside is that when one of the dependencies used is impacted by a security issue then it impact also the security posture of the application itself.
+
+This aspect is referenced in the:
+* [OWASP TOP 10](https://www.owasp.org/index.php/Category:OWASP_Top_Ten_2017_Project) under the point *[A9 - Using Components with Known Vulnerabilities](https://www.owasp.org/index.php/Top_10-2017_A9-Using_Components_with_Known_Vulnerabilities)*.
+* [OWASP Application Security Verification Standard Project](https://www.owasp.org/index.php/Category:OWASP_Application_Security_Verification_Standard_Project) under the section *V14.2 Dependency*.
+
+Due to this context, it's important for a project to ensure that all the third-party dependencies used do not suffer for any security issue **OR** if they contains security issues then the developmment team is aware of that and mitigation measures are in place.
+
+It's highly recommanded to perform automated analysis of the dependencies from the day one of the project. Indeed, if this task is added at the middle or end of the project, it can imply a amount of work to handle all the issues identified that will not be possible to handle by the developmement team since a long time.
```

Comment:
added huge, spelling in **development**, change **since** to **for**

it can imply a **huge** amount of work to handle all the issues identified that will not be possible to handle by the **development** team **for** a long time.

Reply:
Fixed in coming commit.